test/testutils.jl

```diff
@@ -1,23 +1,44 @@
-allapprox(coords₁::C, coords₂::C; kwargs...) where {C<:CRS} =
-  all(ntuple(i -> isapprox(getfield(coords₁, i), getfield(coords₂, i); kwargs...), nfields(coords₁)))
-
-function allapprox(coords₁::C, coords₂::C; kwargs...) where {C<:Cartesian2D}
-  isapprox(coords₁.x, coords₂.x, kwargs...) && isapprox(coords₁.y, coords₂.y, kwargs...)
+svec(coords::Cartesian) = SVector(getfield(coords, :coords))
+svec(coords::CoordRefSystems.Projected) = SVector(coords.x, coords.y)
+
+is_approx_angle(
+  α, β;
+  atol = sqrt(eps(2 * convert(promote_type(Unitful.numtype.((α, β))...), π))),
+) = isapprox(mod2pi(α - β + π), π; atol = ustrip(rad, atol))
```

Comment:
Appreciate if you can comment on the use `mod2pi` and the role of `pi`. Is this expression obtained from case-by-case analysis with antipodal angles?

Reply:
Correct. We technically want 
```julia
isapprox(α - β, 0; atol)
``` 
in a "mod 2π" sense here, and just doing 
```julia
isapprox(mod2pi(α - β), 0; atol)
```
won't give you that because in this way `α - β == 2*k*π - eps < 0` ends up comparing `2π - eps` to `0` instead of `-eps`. I thought fixing this by shifting the origin to π would be pretty neat, but you're prodding now made me realise that an even neater way to write this is 
```suggestion
) = abs(rem2pi(α - β, RoundNearest)) <= ustrip(rad, atol)
```
Will update. 🙏